clonelists/Nintendo - GameCube.json

```diff
@@ -6,10 +6,17 @@
 			"Nintendo - GameCube - NKit ISO",
 			"Nintendo - GameCube - NASOs"
 		],
-		"last updated": "24 May 2021",
+		"last updated": "14 July 2021",
 		"minimum version": "0.89"
 	},
 
+	"categories": {
+		"Ribbit King Plus!": {
```

Comment:
The correct short name here is `Ribbit King Plus! (Bonus Disc)`.

Reply:
I was confused because "Ribbit King Plus!" is the title of the bonus disc itself :|
Do we really need to add the (Bonus Disc) thing?

Reply:
Yep! A short name is different to a group name -- it has to be in order to handle tags like (Disc 1) properly, or to make sure bonus discs don't get tied up with game discs.